app/lib/champaign_queue/clients/sqs.rb

```diff
@@ -6,22 +6,24 @@ class Sqs
       class << self
         # +params+ - The message to send. String maximum 256 KB in size.
         # +delay+  - The number of seconds (0 to 900 - 15 minutes) to delay a specific message.
-        def push(params, delay: 0)
-          new(params, delay).push
+        def push(params, group_id:, delay: 0)
+          new(params, group_id, delay).push
```

Comment:
At this point I'd make params another named parameter:

```diff
- def push(params, group_id:, delay: 0)
+ def push(payload:, group_id:, delay: 0)
```

Since it shows intent (we never push without a payload) and it makes calls to `ChampaignQueue.push` more readable:

```rb
ChampaignQueue.push(
  payload: {type: ..., params: ...}, 
  group_id: "member:#{member.id}"
)
```

Even keep all ChampaignQueue params in their own module: 

```rb
# app/lib/champaign_queue/serializers.rb
module ChampaignQueue
  module Serializers
    extend self

    def update_member(member)
      { type: 'update_member', params: { ... } }
    end

    def subscribe_member(member)
      { type: 'subscribe_member', params: { ... } }
    end
    
    # and so on and so forth
    # def <payload_type>; end
  end
end

# app/controllers/api/stateless/members_controller.rb
ChampaignQueue.push(
  payload: ChampaignQueue::Serializers.update_member(member),
  group_id: "member:#{member.id}"
)
```
Or something along those lines. That way all payload types are in one place which makes it easy to spot the pattern, easy to lookup when debugging, etc.